code/modules/clothing/suits/wiz_robe.dm

```diff
@@ -146,38 +146,44 @@
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
 	resistance_flags = FLAMMABLE
 
-/obj/item/clothing/suit/wizrobe/paper
+/obj/item/clothing/suit/wizrobe/paper //fun fact: because this is a subtype of normal wizard robes, a normal wizard can actually cast spells in this
 	name = "papier-mache robe" // no non-latin characters!
 	desc = "A robe held together by various bits of clear-tape and paste."
 	icon_state = "wizard-paper"
 	item_state = "wizard-paper"
-	var/robe_charge = TRUE
-	actions_types = list(/datum/action/item_action/stickmen)
+	var/timer = 0 //when we'll next be able to use the summon stickman ability
+	var/angry_sticks = TRUE // have you set the robe to make stickmen who WON'T have the neutral faction? note that this WON'T affect stickmen that (whom?) you've already made
+	actions_types = list(/datum/action/item_action/stickmen, /datum/action/item_action/evilstick)
 
 
 /obj/item/clothing/suit/wizrobe/paper/ui_action_click(mob/user, action)
-	stickmen()
+	if(istype(action, /datum/action/item_action/evilstick)) //choose whether or not you want your stickmen to attack neutral mobs, such as other players (but not you, of course!)
+		angry_sticks = !angry_sticks //toggle what angry_sticks is set to
+		to_chat(user, "<span class='warning'>The stickmen you summon will now be more [angry_sticks ? "passive":"aggressive"].</span>")
+		return
+	stickmen() //if you didn't choose the evilstick action, you must have chosen item_action/stickmen; let's try to summon a stickman, then
 
 
 /obj/item/clothing/suit/wizrobe/paper/verb/stickmen()
 	set category = "Object"
-	set name = "Summon Stick Minions"
+	set name = "Summon Stick Minion"
 	set src in usr
 	if(!isliving(usr))
 		return
-	if(!robe_charge)
+	if(timer > world.time)
 		to_chat(usr, "<span class='warning'>\The robe's internal magic supply is still recharging!</span>")
 		return
 
-	usr.say("Rise, my creation! Off your page into this realm!", forced = "stickman summoning")
+	usr.say("Rise, my creation! Off of your page and into this realm!", forced = "stickman summoning")
 	playsound(src.loc, 'sound/magic/summon_magic.ogg', 50, TRUE, TRUE)
-	var/mob/living/M = new /mob/living/simple_animal/hostile/stickman(get_turf(usr))
-	var/list/factions = usr.faction
+	var/mob/living/M = new /mob/living/simple_animal/hostile/stickman/paperrobe(get_turf(usr)) //these guys have half the maximum health of a normal stickman, to make them less cancerous to fight against when spam-summoned
+	var/list/factions = usr.faction.Copy()
+	if(angry_sticks)
+		for(var/F in factions)
+			if(F == "neutral")
+				factions -= F  //this is so that they'll attack other players and NPCs and stuff, but still won't attack the person who made them or each other

```

Comment:
just do `factions -= neutral`, if it doesnt exist it shouldnt freak out.

Reply:
are you absolutely, 100% sure about this?

Reply:
it is just a list.

Reply:
`LAZYREMOVE(factions, "neutral")` should work alright
If the element is not on the list nothing will happen.